packages/dd-trace/src/appsec/remote_config/index.js

```diff
@@ -32,6 +32,18 @@ function enable (config) {
   }
 }
 
+function enableAsm () {
+  if (rc) {
```

Comment:
if rules are set by customer, we shouldn't enable the capability neither subscribe to `ASM`